write-fonts/src/pens.rs

```diff
@@ -270,12 +272,64 @@ impl<'a, T: Pen> Pen for ReverseContourPen<'a, T> {
     }
 }
 
+/// Records commands as [PenCommand]
+pub struct RecordingPen {
+    commands: Vec<PenCommand>,
+}
+
+impl RecordingPen {
+    pub fn new() -> RecordingPen {
+        RecordingPen {
+            commands: Vec::new(),
+        }
+    }
+
+    pub fn commands(&self) -> &Vec<PenCommand> {
```

Comment:
Common practice would be for this to return a slice instead.

Reply:
Fixed, ty